crates/nu-std/tests/logger_tests/test_logger_uses_env_formatting.nu

```diff
@@ -0,0 +1,82 @@
+use std *
+
+def run [ 
+    message: string,
+    level: string,
+    --short
+] {
+    let config_filename = "crates/nu-std/tests/logger_tests/logger_env_test_config.nu"
```

Comment:
i do not quite understand the need for that file :thinking: 

it's an exact clone of the `export-env` block inside `std/log.nu` :confused:

Reply:
Take a closer look — it has different values. Basically, I could not embed overwriting env variables into the command string, so I created a file to `use` after `std` in the `run` command.

Reply:
oooooooh i see what you're testing in this file now :ok_hand: 

let me have a look :wink:

Reply:
mm well, i do not see the value of this test in the end :thinking: 

being able to modify the environment is part of Nushell and anything in an `export-env` block is not frozen :yum: 

it looks quite redundant with the other tests, i'd personally remove it altogether, making the whole test suite, simpler :relieved:

Reply:
Yeah, I can agree with you that I simple overengineered it. Thanks! :D

Reply:
ooooh yeah, no worries :wink: